wiki/Client/Menu_tips/de.md

```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
+- Du kannst überall `Strg` + `T` drücken, um die Symbolleiste zu aktivieren oder zu deaktivieren!
 - Du kannst überall `Strg` + `O` drücken, um auf die Optionen zuzugreifen!
-- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, das Spiel zu pausieren und den Skin zu ändern!
+- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin während des Spiels zu ändern!
 - Mit jedem Update kommen neue Features hinzu. Bleibe immer auf dem Laufenden!
 - Wenn dir die Bedienelemente zu groß oder zu klein sind, versuche, die UI-Skalierung in den Einstellungen anzupassen!
 - Versuche, die `Bildschirmskalierung` anzupassen, um den Bereich der Benutzeroberfläche oder des Gameplays zu ändern, sogar im Vollbildmodus!
-- Was früher "[osu!direct](/wiki/osu!supporter#osu!direct)" war, ist nun wie auf der Webseite für alle Nutzer verfügbar. Du kannst es überall mit `Strg` + `D` aufrufen!
-- In Replays kann durch das Ziehen des Cursors auf der Schwierigkeitsleiste am unteren Rand des Bildschirms gesprungen werden!
+- Was früher "[osu!direct](/wiki/osu!supporter#osu!direct)" war, ist nun wie auf der Webseite für alle Nutzer verfügbar. Du kannst es überall mit `Strg` + `B` aufrufen!
+- In Replays kann durch das Ziehen des Cursors oder durch die linke und rechte Pfeiltaste auf der Fortschrittsleiste am unteren Rand des Bildschirms gesprungen werden!
 - Unterstützung für Multithreading bedeutet, dass selbst bei geringen "FPS" deine Eingaben und Beurteilungen präzise sein werden!
-- Probiere, in der Modauswahl herunterzuscrollen, um einen Haufen neuer spaßiger Mods zu entdecken!
+- Probiere, in der Modauswahl nach rechts zu scrollen, um einen Haufen neuer spaßiger Mods zu entdecken!
 - Die meisten Webinhalte (Profile, Ranglisten etc.) sind über die Symbole in der Symbolleiste direkt im Spiel verfügbar!
 - Erhalte mehr Details, verstecke oder lösche eine Beatmap, indem du mit der rechten Maustaste auf ihr Dialogfeld in der Songauswahl klickst!
 - Alle Löschoperationen sind bis zum Verlassen des Spiels nur vorübergehend. Stelle versehentlich gelöschte Inhalte in den Wartungseinstellungen wieder her!
 - Probiere das "Playlists"-System aus, das Nutzer ihre eigenen, bleibenden Bestenlisten kreieren lässt!
 - Mit `Strg` + `F11` kannst du Statistiken über Frames und Threads erhalten!
 - Werfe einen Blick unter die Haube der Leistungsindikatoren und aktiviere die ausführliche Leistungsprotokollierung mit `Strg` + `F2`!
+- Während eines Replays kannst du mit `Leertaste` pausieren!
+- Die meisten der Tastaturkürzel im Spiel sind konfigurierbar und auf die eigenen Bedürfnisse anpassbar. Schlage die Tastenbelegungen in den Eingabeeinstellungen nach!
+- Wenn das HUD im Spiel verborgen ist, kannst du `Strg` halten, um es vorübergehend zu sehen!
+- Das HUD im Spiel kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
```

Comment:
```suggestion
- Das Spieldesign kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
```
🤔

Reply:
Both are not very descriptive.. Is using `Gameplay` a problem? I think it's a common enough word, and `Gameplay-HUD` would be obvious to most I assume.

Reply:
```suggestion
- Das Gameplay-HUD kann mit dem Skin-Editor angepasst werden. Öffne ihn jederzeit mit `Strg` + `Shift` + `S`!
```